server/game/cards/11-TCaR/Mausoleum.js

```diff
@@ -0,0 +1,22 @@
+const DeedCard = require('../../deedcard.js');
+
+class Mausoleum extends DeedCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            title: 'Mausoleum',
+            cost: ability.costs.bootSelf(),
+            when: {
+                onCardAced: event =>
+                    !event.card.hasOneOfKeywords(['token', 'abomination']) && event.card.getType() === 'dude'
+            },
+            message: context => this.game.addMessage('{0} uses {1} to have {1} gain a permanent CP', context.player, this),
```

Comment:
Not sure we need the name of the card twice since it is right afterwards.
```suggestion
            message: context => this.game.addMessage('{0} uses {1} to have it gain a permanent CP', context.player, this),
```